pytest_ethereum/_utils/linker.py

```diff
@@ -50,19 +55,21 @@ def create_latest_block_uri(w3: Web3, tx_receipt: TxReceipt) -> URI:
 
 
 def insert_deployment(
-    package: Package,
+    manifest: Manifest,
+    latest_block_uri: URI,
```

Comment:
Using the term `latest` here might not be ideal since:

1. The latest block could end up getting uncle'd in a re-org
2. Our code should probably not default to the very latest block, but ideally something like ~5 blocks back.  Also take into account that we can't go back further than the block where the deployment happened.